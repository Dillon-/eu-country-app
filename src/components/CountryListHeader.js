import React from 'react';
import { Text } from 'react-native';
import PropTypes from 'prop-types';
import EStyleSheet from 'react-native-extended-stylesheet';

const CountryListHeader = ({ title }) => {
  return (
    <Text style={styles.text}>{title}</Text>
  )
};

CountryListHeader.propTypes = {
  title: PropTypes.string.isRequired,
};

const styles = EStyleSheet.create({
  text: {
    paddingHorizontal: 20,
    paddingTop: 20,
    paddingBottom: 10,
    color: 'white',
    fontWeight: '800',
    fontSize: 22,
  }
});

export default CountryListHeader;