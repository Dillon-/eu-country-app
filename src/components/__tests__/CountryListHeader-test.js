import React from 'react';
import renderer from 'react-test-renderer';

import CountryListHeader from '../CountryListHeader';

it('renders a CountryListHeader', () => {
  const component = renderer.create(
    <CountryListHeader
      title="Europe"
    />
  );
  expect(component).toMatchSnapshot();
});