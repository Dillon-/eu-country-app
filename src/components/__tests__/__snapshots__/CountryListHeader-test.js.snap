// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a CountryListHeader 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={undefined}
>
  Europe
</Text>
`;
